qa/tasks/deepsea.py

```diff
@@ -1226,12 +1226,12 @@ def _run_stage_3(self):
             'ceph_cluster_status.sh',
             )
         self.__ceph_health_test()
-        self.log.info("Entering temporary post-Stage-3 sleep (3 minutes). "
-                      "Purpose is to avoid transient failures that started "
-                      "appearing with ceph 14.2.9. Once the DeepSea suite "
-                      "is shown to pass reliably on 14.2.9+ without this sleep, "
-                      "it can be removed.")
-        time.sleep(180)
+        # self.log.info("Entering temporary post-Stage-3 sleep (3 minutes). "
+        #               "Purpose is to avoid transient failures that started "
+        #               "appearing with ceph 14.2.9. Once the DeepSea suite "
+        #               "is shown to pass reliably on 14.2.9+ without this sleep, "
+        #               "it can be removed.")
+        # time.sleep(180)
```

Comment:
I think you can just delete these lines instead of commenting them out. If needed, the code can be retrieved at any time by simply reverting this commit.

Reply:
ah, ok. Done